src/main/resources/db/migration/V044__Add_validation_warnings_column.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE scannable_items
+ADD COLUMN ocrValidationWarnings VARCHAR(1000)[] NULL;
```

Comment:
not sure if varchar(1000) is sufficient or should be keep it bit more ?

Reply:
How about 5k?

Reply:
Seems more than enough for a single warning message to me

Reply:
Sample 1000 char string:
>mi quis hendrerit dolor magna eget est lorem ipsum dolor sit amet consectetur adipiscing elit pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas integer eget aliquet nibh praesent tristique magna sit amet purus gravida quis blandit turpis cursus in hac habitasse platea dictumst quisque sagittis purus sit amet volutpat consequat mauris nunc congue nisi vitae suscipit tellus mauris a diam maecenas sed enim ut sem viverra aliquet eget sit amet tellus cras adipiscing enim eu turpis egestas pretium aenean pharetra magna ac placerat vestibulum lectus mauris ultrices eros in cursus turpis massa tincidunt dui ut mi quis hendrerit dolor magna eget est lorem ipsum dolor sit amet consectetur adipiscing elit pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas integer eget aliquet nibh praesent tristique magna sit amet purus gravida quis blandit turpis cursus in hac habitasse platea dictumst quisque sagittis purusus

Reply:
and that's per warning